cmd/evm/runner.go

```diff
@@ -74,37 +74,40 @@ func readGenesis(genesisPath string) *core.Genesis {
 	return genesis
 }
 
-type execStats struct {
-	time           time.Duration // The execution time.
-	allocs         int64         // The number of heap allocations during execution.
-	bytesAllocated int64         // The cumulative number of bytes allocated during execution.
+type ExecStats struct {
```

Comment:
Does this need to be exported? I don't see why

Reply:
Ah I think I got confused because non-exported members of structs are not serialized in JSON so I assumed that the type also had to be exported 🤷‍♂️

Reply:
Wait but how can `StateTestResult` have a public field with a private type...